test/chi/js/popover.pug

```diff
@@ -0,0 +1,61 @@
+---
+title: Popover
+---
+script(src='/js/chi.js')
+
+- var moreSides = ["top", "top-start", "top-end", "right", "right-start", "right-end", "bottom", "bottom-start", "bottom-end", "left", "left-start", "left-end"];
+
+button#popover-1b.chi-button.popover-btn(data-target='#popover-in-html',data-cy='show-popover') Popover
+
+section#popover-in-html.chi-popover(aria-modal='true', role='dialog', aria-label='Popover title')
+  header.chi-popover__header
+    h2.chi-popover__title Popover title
+  .chi-popover__content
+    p.chi-popover__text Popover content
+
+each side in moreSides
+    div(class=`test-more-${side} -position--relative`,data-cy=`test-more-${side}`)
+        button.chi-button.popover-btn(data-position=`${side}`, data-popover-content='<header class="chi-popover__header"><h2 class="chi-popover__title">Popover positioning</h2></header><div class="chi-popover__content"><p class="chi-popover__text">Popover Content.</p></div>')=`${side} popover`
+
+.parent(data-cy='auto-hide')
+
+  button#popover-6.chi-button(data-cy='popover-6',data-popover-content='<header class="chi-popover__header"><h2 class="chi-popover__title">Popover title</h2></header><div class="chi-popover__content"><p class="chi-popover__text">Click outside. I will disappear!</p></div>') Auto hide
+  script.
```

Comment:
Can we move all the JS to the bottom of the file?
Let's consolidate all the scripts together, please.